utilities/qneg.meta

```diff
@@ -133,13 +133,13 @@
   type  = scheme
 [ mpi_communicator ]
   standard_name = mpi_communicator
-  units = 1
+  units = index
   type = integer
   dimensions = ()
   intent = in
 [ rootprocid ]
```

Comment:
Isn't this mpi_root (based on what is in the CCPP standard dictionary)?

Reply:
I think you're looking at the local name - the standard name is indeed mpi_root

Reply:
Yup - you are correct